common/buildcraft/builders/TileQuarry.java

```diff
@@ -205,7 +205,7 @@ public void updateEntity() {
 			// In this case, since idling() does it anyway, we should return.
 			return;
 		} else if (stage == Stage.MOVING) {
-			int energyUsed = this.getBattery().useEnergy(20, (int) Math.ceil(20 + getBattery().getEnergyStored() / 10), false);
+			int energyUsed = this.getBattery().useEnergy(20, (int) Math.ceil(20D + (double) getBattery().getEnergyStored() / 10), false);
```

Comment:
Casting to double, 10.0 and 10D in bytecode is identical. I check it in BytecodeOutline.